templates/portals.example.yml

```
@@ -95,6 +95,29 @@ title_filter:
+  # date_posted: filter by posting age. Options: 24, Week, Month
+  date_posted: Week
+  # experience_level: Options: Entry-level, Senior, Manager, Director, Executive
+  # experience_level: [Senior, Manager]
+  max_results_per_search: 25
+  delay_between_pages_ms: [3000, 8000]
+  delay_between_searches_ms: [5000, 15000]
+  # append "remote" to keywords to filter for remote jobs. Jobs near your location will be included otherwise.
+  keywords:
+    - AI Engineer
+    - Software Engineer remote
+  # employer_blocklist: companies to skip (case-insensitive, partial match)
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)